internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -206,26 +184,32 @@ func (w *sessionWorkflow) initEnvModule(cinfos map[string]connInfo) error {
 
 func integrationModulePrefix(name string) string { return fmt.Sprintf("__%v__", name) }
 
-func (w *sessionWorkflow) initConnections(ctx workflow.Context) (map[string]connInfo, error) {
+func (w *sessionWorkflow) initConnections(wctx workflow.Context) (map[string]connInfo, error) {
+	// While theoretically all this code is reaching external systems for integrations, but since
+	// all the integrations are currently builtin, the operations are instantanious.
+	// No need to use activities and such right now.
```

Comment:
1. Thanks for this comment!
2. See fixes for typos
3. Missing: what will need to change in the future when this assumption breaks?
4. We won't remember this function when this assumption breaks, add a TODO with a ticket?

```suggestion
	// In theory, this code is reaching external systems for integrations, but since
	// all the integrations are currently bundled with the AK binary, the operations
	// are instantaneous. No need to use activities and such right now.
```

Reply:
fixed the typo. i'm going to kill this code soon though - we are going to kill all methods on intergrations.